slack_today_i_did/bot_file.py

```diff
@@ -73,6 +89,10 @@ def parse_direct_message(self, message):
                     report.add_response(name, text)
                     self.send_message(name, 'Thanks!')
 
+        if self.sessions.has_running_session(user):
```

Comment:
:art: :art: looking like a case for `register_step` (just a general commentary, not expecting to be addressed with this PR)